### ISSUE TITLE
logging is still too sketchy

### ISSUE THREAD

**Issue by yegor256** [objectionary/jeo-maven-plugin#431]
I'm getting this information when I run `opeo` plugin:

```
[INFO] --- opeo:0.1.2:compile (default-cli) @ benchmark ---
[INFO] Compiling EO sources from after/generated-sources/ineo-fuse
[INFO] Saving new compiled EO sources to after/generated-sources/opeo-compile
[INFO] Compiled after/generated-sources/opeo-compile/org/eolang/benchmark/App.xmir (6Kb)
[INFO] Compiled after/generated-sources/opeo-compile/org/eolang/benchmark/BA.xmir (9Kb)
[INFO] Compiled after/generated-sources/opeo-compile/org/eolang/benchmark/B.xmir (8Kb)
[INFO] Compiled after/generated-sources/opeo-compile/org/eolang/benchmark/A.xmir (7Kb)
[INFO] Compiled after/generated-sources/opeo-compile/org/eolang/benchmark/Main.xmir (14Kb)
[INFO] Compiled after/generated-sources/opeo-compile/org/eolang/benchmark/F.xmir (2Kb)
[INFO] Compiled 6 sources
```

Would be great to have similar information from `jeo`, especially about the location of sources, output, and a summary at the end about total count of compiled sources. Also, would be nice to print _two_ lines for each file: `Compiling A.xmir (5Kb)...` and then, when finished, `Compiled A.xmir (6Kb) in 230ms`. The first line may be moved to `DEBUG` class, when #429 is fixed.

**Comment by 0pdd** [comment]
@yegor256 2 puzzles [#447](https://github.com/objectionary/jeo-maven-plugin/issues/447), [#448](https://github.com/objectionary/jeo-maven-plugin/issues/448) are still not solved.

**Comment by 0pdd** [comment]
@yegor256 2 puzzles [#447](https://github.com/objectionary/jeo-maven-plugin/issues/447), [#452](https://github.com/objectionary/jeo-maven-plugin/issues/452) are still not solved; solved: [#448](https://github.com/objectionary/jeo-maven-plugin/issues/448).

**Comment by 0pdd** [comment]
@yegor256 2 puzzles [#452](https://github.com/objectionary/jeo-maven-plugin/issues/452), [#454](https://github.com/objectionary/jeo-maven-plugin/issues/454) are still not solved; solved: [#447](https://github.com/objectionary/jeo-maven-plugin/issues/447), [#448](https://github.com/objectionary/jeo-maven-plugin/issues/448).

**Comment by 0pdd** [comment]
@yegor256 the puzzle [#452](https://github.com/objectionary/jeo-maven-plugin/issues/452) is still not solved; solved: [#447](https://github.com/objectionary/jeo-maven-plugin/issues/447), [#448](https://github.com/objectionary/jeo-maven-plugin/issues/448), [#454](https://github.com/objectionary/jeo-maven-plugin/issues/454).

**Comment by 0pdd** [comment]
@yegor256 all 4 puzzles are solved here: [#447](https://github.com/objectionary/jeo-maven-plugin/issues/447), [#448](https://github.com/objectionary/jeo-maven-plugin/issues/448), [#452](https://github.com/objectionary/jeo-maven-plugin/issues/452), [#454](https://github.com/objectionary/jeo-maven-plugin/issues/454).

**Comment by volodya-lombrozo** [comment]
@yegor256 Added all the logs which you mentioned. Could you check `0.3.0` version, please?
